translations/tgl.po

```diff
@@ -745,7 +745,7 @@ msgstr "Malalaro ang mga level na ginawa ng komunidad sa #ffff00ff\"pangkomunida
 
 #: ../client/application/game/ui/screens/screen_select_single_player_level.cpp:78
 msgid "More info on github.com/jyaif/ppl-utils"
-msgstr "Iba pang impormasyon sa github.com/jyaif/ppl-utils"
+msgstr "(Iba pang impormasyon sa #ffff00ffgithub.com/jyaif/ppl-utils#ffffffff)"
```

Comment:
I'm not sure yellow is a nice link color, it might be confused with the color for level types. 
What you could do is use a more bluish color like Russian does, the color code is `#0080ffff`.

If you think yellow is fine, you can keep it. It's not mandatory to change the color!

Reply:
Yeah, just edit the file again and the PR will update